package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Convert JSON schema to Google BigQuery schema",
   "main": "src/index.js",
   "scripts": {
-    "test": "mocha test"
+    "test": "mocha test",
+    "lint": "eslint ."
```

Comment:
This should of been done in a separate PR.

Don't forget this is not a HX project so changing/applying linting on someone else project is a bit intrusive.